test/functional/core/job_spec.lua

```diff
@@ -138,7 +139,7 @@ describe('jobs', function()
   end)
 
   it('allows interactive commands', function()
-    nvim('command', "let j = jobstart(['cat', '-'], g:job_opts)")
+    nvim('command', "let j = jobstart(['"..nvim_cat.."', '-'], g:job_opts)")
```

Comment:
Doesn't `foo.cmd` require a shell? So I would not expect this to work, the `jobstart({list})` form skips the shell. 

That's one reason why including `cat.exe` was important. 

And changing these invocations to use the shell form of `jobstart({string})` changes the nature of the tests, so we don't want to do that.

Reply:
You're right. I'll keep the list form and run the batchfile with cmd.exe directly. It should be fine if the tempfile doesn't have to be escaped.